# Robot Configuration Parameters
acc_lim_x: 2.5
acc_lim_y: 2.5
acc_lim_th: 3.2
max_vel_x: 5
min_vel_x: 0.1
max_rotational_vel: 1.75
min_in_place_rotational_vel: 0.4
escape_vel: -0.1
holonomic_robot: false
publish_cost_grid_pc: true

# Goal Tolerance Parameters
xy_goal_tolerance: 0.1
yaw_goal_tolerance: 0.05

# Forward Simulation Parameters
sim_time: 1.7
sim_granularity: 0.025

vx_samples: 15
vtheta_samples: 20

# Trajectory Scoring Parameters
goal_distance_bias: 5
path_distance_bias: 2
occdist_scale: 5

heading_lookahead: 0.325
dwa: true

# Oscillation Prevention Parameters
oscillation_reset_dist: 20

